components/backup-stream/src/endpoint.rs

```diff
@@ -223,6 +227,13 @@ where
         self.pool.block_on(async move {
             // TODO: also pause the task using the meta client.
             let err_fut = async {
+                self.pd_client
+                    .update_service_safe_point(
```

Comment:
how to remove this safe point, when user want to remove the task and start it over.